docs/install-guide.md

```diff
@@ -1,21 +1,17 @@
 # OpenChoreo Installation
 
-This guide walks you through the installation and setup of OpenChoreo on a multi-cluster Kubernetes environment. 
-The process involves creating and configuring two Kubernetes clusters: one for the **Control Plane** and another for the **DataPlane**. 
-After configuring the clusters, you will install OpenChoreo using Helm, verify the installation, and install the choreoctl CLI tool to manage OpenChoreo components.
+This guide walks you through installing and setting up OpenChoreo on a single Kubernetes cluster using kind. 
+The *Control Plane* and *Data Plane* components will run on the same cluster. This is ideal for local development or trying out OpenChoreo quickly.
 
-By the end of this guide, you'll have a fully functional OpenChoreo deployment running on a multi-cluster setup.
+You'll start by creating a compatible Kubernetes cluster with Cilium, install OpenChoreo using CP and DP Helm charts, and set up the choreoctl CLI.
 
+Additionally, the guide provides options to expose the OpenChoreo external gateway service to your host machine, enabling seamless access to the components you create.
 
-## Create Compatible Kubernetes Clusters
-
-You need two Kubernetes clusters: one for the Control Plane and one for the Data Plane. The Data Plane cluster should have Cilium installed for compatibility with OpenChoreo.
-
-If you don’t have compatible Kubernetes clusters yet, you can create them using the following guide on your local machine.
+If you don't have a compatible kubernetes cluster, you can create one of following in your local machine and start testing.
```

Comment:
since we removed the topic ## Create Compatible Kubernetes Clusters, I think we don't need this sentence. At this point user knows he is going to create a new kubernetes cluster, as we mentioned in line 8